mne/surface.py

```diff
@@ -1917,20 +1917,20 @@ def _marching_cubes(image, level, smooth=0, fill_hole_size=None, use_flying_edge
             f"{level.size} elements"
         )
 
+    image_shape = image.shape
+    order = "C" if image.flags.c_contiguous else "F"
+    image = deepcopy(image)  # don't modify original
```

Comment:
Better than this I think would be to do `image = np.ascontiguousarray(image)`, it guarantees C-contiguity so you don't need any more conditional. Also arrays can be neither C nor F contiguous (e.g., if there is a non-unity stride) so the current code would be broken for such arrays

Reply:
I tried that and it didn't work for the tumor test case. Want to give it a quick look?

Reply:
Yep, you need the tumor nrrd file and to pip install pynrrd from the original issue too. If it resembles a blob, then it worked, otherwise you get a cube of dots.